Tests/SentryProfilerTests/SentryTraceProfilerTests.swift

```diff
@@ -428,9 +428,9 @@ private extension SentryTraceProfilerTests {
         }
     }
 
-    func printTimestamps(entries: [[String: Any]]) -> [NSString] {
-        entries.reduce(into: [NSString](), { partialResult, entry in
-            partialResult.append(entry["elapsed_since_start_ns"] as! NSString)
+    func printTimestamps(entries: [[String: Any]]) throws -> [NSString] {
+        try entries.reduce(into: [NSString](), { partialResult, entry in
+            partialResult.append(try XCTUnwrap(entry["elapsed_since_start_ns"] as? NSString))
```

Comment:
Does this really need to throw? Or you add it to add it just to comply with XCTUnwrap?

Can we use this?
```Swift
    func printTimestamps(entries: [[String: Any]]) -> [String] {
        return entries.compactMap { $0["elapsed_since_start_ns"] as? String }
    }
```

Reply:
oh good point, i like that. then an xctunwrap will work on the result later 👍🏻